docs/development/pigs/pig-001.rst

```diff
@@ -0,0 +1,120 @@
+Please make your proposal clearly and keep the initial proposal short.
+If more information is needed, people that will review it will ask for it.
+
+Writing a PIG doesn't mean you have to implement it. That said, we expect that
+most PIGs will propose something that requires developments where someone has
+the intention to implement it within the near future (say within the next year).
+But it's not required, e.g. if you have a great idea or vision for Gammapy
+that requires a lot of development, without the manpower to execute the idea,
+writing a PIG could be a nice way to share this idea in some detail, with the
+hope that in collaboration with others it can eventually be realised.
+
+PIG review
+==========
+
+PIG review happens on the pull request on Github.
+
+When a PIG is put up, an announcement with a link to the pull request should
+be sent both to the Gammapy mailing list and the Gammapy coordinator list.
+
+Anyone is welcome to review it and is encouraged to share their thoughts
+in the discussion!
+
+Please note that Github hides inline comments after they have been edited,
+so we suggest that you use inline comments for minor points like spelling mistakes only.
+Put your main feedback as normal comments in the "Conversation" tab,
+so that for someone reading the discussion later they will see your comment directly.
+
+The final decision on any PIG is made by the Gammapy coordination committee.
+We expect that in most cases, the people participating in the PIG review will
+reach a consensus and the coordination committee will simply follow the outcome
+of the public discussion. But in unusual cases where disagreement remains,
+the coordination committee will talk to the people involved in the discussion
+with the goal to reach consensus or compromise, and then make the final decision.
+
+PIG status
+==========
+
+PIGs can have a status of:
+
+* "draft" - in draft status, either in the writing or dicussion phase
+* "withdrawn" - withdrawn by the author
+* "accepted" - accepted by the coordination committee
+* "rejected" - rejected by the coordination committee
+
+Final remarks
+=============
+
+This PIG leaves some points open. Specifically we don't say whether
+PIGs are accepted and merged before or after the actual implementation is done,
+and whether PIGs are fixed or live documents, i.e. whether they are edited
+when things change. We also don't specify how the Gammapy coordination committee
+should make they decision if opinions from the people in that group differ.
+This is intentional. We want to keep the process flexible
+and first gain some experience. The goal of PIGs is to help the Gammapy developer
+team to be more efficient, not to have a rigid or bureaucratic process.
+
+.. _PEPs: https://www.python.org/dev/peps/pep-0001/
+.. _NEPs: https://docs.scipy.org/doc/numpy-dev/neps/index.html
+.. _APEs: https://github.com/astropy/astropy-APEs
+.. _JEPs: https://github.com/jupyter/enhancement-proposals
```

Comment:
The [APE that introduced APEs](https://github.com/astropy/astropy-APEs/blob/master/APE1.rst) - you might call it MOAAPE (mother of all APEs) - has a last section summarizing the discussion, which in this case apparently was very short. But in general I think it's nice to have such a section, especially to avoid that new people attack a PIG with the same arguments that many others have put forward before. For this PIG there won't be much discussion, but I suggest to put in this PIG that other PIGs should have such a paragraph.

Reply:
Agreed. I've added the suggestion to put a "decision rationale" section at the end of the PIG, as they do for APEs.